examples/eks/eks-cluster.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: <eks_cluster_name>
+  region: <eks_region>
+
+nodeGroups:
+
+  - name: scylla-pool
+    instanceType: i3.2xlarge
+    desiredCapacity: 3
+    labels:
+      pool: "scylla-pool"
+    taints:
+      role: "scylla-clusters:NoSchedule"
+    ssh:
+      allow: true
+    kubeletExtraConfig:
+      cpuManagerPolicy: static
+    availabilityZones:
+      - <eks_zone>
+
+  - name: cassandra-stress-pool
+    instanceType: c4.2xlarge
+    desiredCapacity: 4
+    labels:
+      pool: "cassandra-stress-pool"
+    taints:
+      role: "cassandra-stress:NoSchedule"
+    ssh:
+      allow: true
+    preBootstrapCommands:
+      - curl -o /etc/yum.repos.d/scylla.repo -L http://repositories.scylladb.com/scylla/repo/3e3ec6ec-92b2-46ec-b64f-5b2338bc47de/centos/scylladb-4.0.repo
+      - yum install -y scylla-tools
+
+  - name: monitoring-pool
+    instanceType: i3.large
+    desiredCapacity: 1
+    labels:
+      pool: "monitoring-pool"
+    ssh:
+      allow: true
+    preBootstrapCommands:
+      - blkdiscard /dev/nvme0n1
+      - mkfs.xfs -f /dev/nvme0n1
+      - mkdir -p /mnt/raid-disks/disk0
+      - mount -t xfs -o noatime /dev/nvme0n1 /mnt/raid-disks/disk0
```

Comment:
This should not be part of the example, no? This is not something we want users to copy and past and edit.
Should it be a DaemonSet or a Init Container?

Reply:
Yep, should be. We have Deamonset configuring Scylla nodes, another one for monitoring seems like a good idea. I'll add it.